### PR TITLE
Bug 1276208: Upgrade nodejs in dockerfiles and node dependencies

### PR DESCRIPTION
**Pull request by pmac**
* Upgrade to latest less
* Switch to yuglify for CSS and JS minification
* Switch to unminified snap.svg as the minified version breaks uglify-js
* Change the base image to quay.io/mozmar/ubuntu-slim-python

### CODE REVIEW

**Comment by pmac** [comment]
@jgmize r?

This will require a good bit of testing, but it's all working locally with my OSX node v6 and in docker. It may conflict with your docker dev work, but hopefully we can resolve that.

**Comment by jgmize** [comment]
Failed during integration tests: https://ci.us-west.moz.works/job/bedrock_integration_tests_runner/7527/console

reverted, will submit new PR with fix

**Comment by pmac** [comment]
Doh! Thanks Josh.

**Comment by pmac** [comment]
Aww man. I swear I'd fixed that and added python-setuptools to the install line. How'd that go away :(

File: docker/dockerfiles/bedrock_base [hunk 1]
```diff
@@ -11,11 +11,10 @@ CMD ["./docker/run-prod.sh"]
 RUN adduser --uid 1000 --disabled-password --gecos '' --no-create-home webdev
 
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends python2.7 libpython2.7 gettext build-essential libxml2-dev libxslt1.1 libxslt1-dev python-dev python-pip nodejs npm git postgresql-client libpq-dev
-RUN update-alternatives --install /usr/bin/node node /usr/bin/nodejs 10
-
-COPY ./bin/pipstrap.py /app/bin/pipstrap.py
-RUN ./bin/pipstrap.py
+    apt-get install -y --no-install-recommends \
+                    gettext build-essential python-dev python-setuptools \
```
Comment (pmac) [inline docker/dockerfiles/bedrock_base]: Yeah, it's right here! @jgmize could it be that said failed just just didn't use a newly built base image?

**Pull request merged**